File: app/src/main/java/info/nightscout/androidaps/plugins/pump/medtronic/data/dto/ClockDTO.java
package info.nightscout.androidaps.plugins.pump.medtronic.data.dto;

import org.joda.time.LocalDateTime;

/**
 * Created by andy on 2/27/19.
 */

public class ClockDTO {

    public LocalDateTime localDeviceTime;

    public LocalDateTime pumpTime;

    public int timeDifference; // s (pump -> local)
}

File: pump/medtronic/src/main/kotlin/app/aaps/pump/medtronic/data/dto/ClockDTO.kt
package app.aaps.pump.medtronic.data.dto

import org.joda.time.LocalDateTime

/**
 * Created by andy on 2/27/19.
 */
class ClockDTO(
    var localDeviceTime: LocalDateTime,
    var pumpTime: LocalDateTime
) {

    // var localDeviceTime: LocalDateTime? = null
    // var pumpTime: LocalDateTime? = null
    var timeDifference = 0
}